components/probe-response-time/index.tsx

```diff
@@ -3,24 +3,36 @@ import {
   InputHTMLAttributes,
   useState,
   useContext,
+  useEffect,
 } from 'react';
 import { TextInput, Checkbox } from '..';
 import { ProbeContext } from '../../contexts/probe-context';
+import { parseAlertStringTime } from '../../utils/parse-alert-string-time';
 
 export interface ProbeResponseTimeProps
   extends Omit<InputHTMLAttributes<HTMLInputElement>, 'className'> {
   probeId: string;
+  alert: string | undefined;
```

Comment:
`alert?: string` equals `alert: string | undefined`.

Reply:
My bad. Should be fixed.